src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,23 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String manufacturer;
+    private String model;
+    private String color;
+    private int maxSpeed;
+    private Engine engine;
 
+    public Car(String manufacturer, String model, String color, int maxSpeed, Engine engine) {
+        this.manufacturer = manufacturer;
+        this.model = model;
+        this.color = color;
+        this.maxSpeed = maxSpeed;
+        this.engine = engine;
+    }
+
+    @Override
+    public Car clone() throws CloneNotSupportedException {
+
+        return new Car(manufacturer, model, color, maxSpeed, engine);
```

Comment:
what do you think, will engine be cloned? what if we change engine name on the original car?

also, blank line on line 20 is redundant

Reply:
tested, found mistake, thank you
fixed